flixel/addons/effects/FlxEffectTrail.hx

```diff
@@ -0,0 +1,188 @@
+package flixel.addons.effects;
+
+import flixel.math.FlxMath;
+import flixel.math.FlxPoint;
+import flixel.util.FlxColor;
+import flixel.util.FlxDestroyUtil;
+import openfl.display.BitmapData;
+import openfl.geom.ColorTransform;
+import openfl.geom.Matrix;
+import openfl.geom.Point;
+using flixel.util.FlxArrayUtil;
+
+/**
+ * This creates a trail copying the bitmapData many times.
+ * 
+ * @author adrianulima
+ */
+class FlxEffectTrail implements IFlxEffect
```

Comment:
What advantage does this have over the already existing `FlxTrail` and `FlxTrailArea`? Those are fairly flexible already, and having 3 separate implementations for essentially the same thing seems like overkill.